### PR TITLE
add static installation

### PR DESCRIPTION
**Pull request by felix-lambert**
Add some installation directive for static application.

### CODE REVIEW

File: source/index.md [hunk 1]
```diff
@@ -946,6 +946,40 @@ Coming soon...
...
+<li>
+2. Create an index.html file at the root of your folder. You can use any single page app framework you want (angularjs, reactjs, backbonejs), or just static html/css pages.
+</li>
+<li>
+3. When you're happy about your work, you can publish it on [github](https://github.com/).
+</li>
+<li>
+4. Then you can go to the home of your Cozy, and at the bottom of the application manager, just type the URL of the repository of your wrapper, click install, wait a few seconds and enjoy !
+</li>
+</ul>
```
Comment (jsilvestre) [inline source/index.md]: To make a list in markdown, simply use "*" at the begining of the line

File: source/index.md [hunk 2]
```diff
@@ -946,6 +946,40 @@ Coming soon...
 We like to experiment in the Node.js ecosystem, and we're promoting single page applications as the way to buld rich experience for the web. This tutorial doesn't cover that aspect. If you want to learn more, please check our [tutorial](https://blog.cozycloud.cc/post/2015/09/29/Build-and-share-your-single-page-app-with-React%2C-Node-and-Pouchdb%2C-part-I)!
 
 
+## Install a static application into your cozy platform
+
+To install a static application into your cozy platform, you need to do the following steps :
+
+<br style="clear: both;" />
```
Comment (jsilvestre) [inline source/index.md:951]: Is it useful here?

File: source/index.md [hunk 3]
```diff
@@ -946,6 +946,40 @@ Coming soon...
 We like to experiment in the Node.js ecosystem, and we're promoting single page applications as the way to buld rich experience for the web. This tutorial doesn't cover that aspect. If you want to learn more, please check our [tutorial](https://blog.cozycloud.cc/post/2015/09/29/Build-and-share-your-single-page-app-with-React%2C-Node-and-Pouchdb%2C-part-I)!
 
 
+## Install a static application into your cozy platform
+
+To install a static application into your cozy platform, you need to do the following steps :
```
Comment (jsilvestre) [inline source/index.md]: I think the right term is "package" not "install" in this case.

**Comment by jsilvestre** [comment]
Thank you for the PR, I only have a few comments that I'd like you to answer/address!

File: source/index.md (hunk 2 above)
Comment (lemelon) [inline source/index.md]: I think that it is IMHO

File: source/index.md [hunk 4]
```diff
@@ -946,6 +946,29 @@ Coming soon...
...
+  {    
+    "name": "cozy-[name of your app]", 
+    "description": "[Description of your app : it will appear on your cozy-home platform]",
+    "cozy-type": "static", 
+    "cozy-displayName": "[The name you want displayed on your cozy-home platform]", 
+    "icon-path": "[the path of the icon you want to use for your static app ('path/to/icon.png')"
+  }
+  ```
+  
+To deploy a static application into your cozy platform, you need to do the following steps :
```
Comment (jsilvestre) [inline source/index.md:959]: - typo: "steps:" (no space before ":")

File: source/index.md [hunk 5]
```diff
@@ -946,6 +946,29 @@ Coming soon...
...
+
+* You need to create a npm package (package.json) into the root of your folder. The name in package.json should be the same as the name of the github repo. Don't forget to put “cozy” in front of name. And the most important of all is to add "cozy-type": "static" for specifying to the controller that it is a static app. If your app is not static, you don't need this line.<br/>
```
Comment (jsilvestre) [inline source/index.md]: > You need to create a npm package

I am concerned that people will think about a published NPM package. Maybe we could use "NPM package manifest" to precisely target the package.json file and clear ambiguity.  What do you think?

> Don't forget to put “cozy” in front of name.

It is not required by the platform, why would you like to push it?

> "cozy-type": "static" for specifying to the controller that it is a static app

Can you "soft-quote" the "code"? Like this: `"cozy-type": "static"`, using `.

File: source/index.md [hunk 6]
```diff
@@ -946,6 +946,29 @@ Coming soon...
...
+If everything is going fine, cozy-controller will be able to clone the repository and install it. If the application is a dynamic app that has his own server, the controller will be able to run it for you. If the application is static, the controller just needs to install it. <br/>
+Your app could be broken if the controller did'nt manage to do one of these operations : clone it, install it or launch it. If for example your application has a package.json that has some syntax error, or your file that launches the app has'nt been found, your application will be broken.
```
Comment (jsilvestre) [inline source/index.md]: typo: "didn't"

Comment (jsilvestre) [inline source/index.md]: I would remove anything not related to static app, which will be described in another section. But I'm being picky here, so don't touch it if you don't feel like it.

File: source/index.md [hunk 7]
```diff
@@ -946,6 +946,29 @@ Coming soon...
...
+
+* You need to create a npm package (package.json) into the root of your folder. The name in package.json should be the same as the name of the github repo. Don't forget to put “cozy” in front of name. And the most important of all is to add "cozy-type": "static" for specifying to the controller that it is a static app. If your app is not static, you don't need this line.<br/>
+If everything is going fine, cozy-controller will be able to clone the repository and install it. If the application is a dynamic app that has his own server, the controller will be able to run it for you. If the application is static, the controller just needs to install it. <br/>
+Your app could be broken if the controller did'nt manage to do one of these operations : clone it, install it or launch it. If for example your application has a package.json that has some syntax error, or your file that launches the app has'nt been found, your application will be broken. 
+* Create an index.html file at the root of your folder. You can use any single page app framework you want (angularjs, reactjs, backbonejs), or just static html/css pages.
```
Comment (jsilvestre) [inline source/index.md:964]: Can you please add links to Angular.js, React.js and Backbone.js official page?

**Comment by jsilvestre** [comment]
Thank you for the new commits, it's almost perfect! Feel free to discuss my points, I'm challenging your work, not enforcing my ideas.

File: source/index.md (hunk 5 above)
Comment (felix-lambert) [inline source/index.md]: Yes i agree.

**Comment by jsilvestre** [comment]
Perfect, thanks!

**Pull request merged**